azure-pipelines.yml

```diff
@@ -37,3 +37,8 @@ steps:
     versionSpec: $(nodejs_version)
 - script: npm run ci-pipeline
   displayName: Build and test
+- bash: bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/master/synopsys-detect) ci-build digitalasset_ex-daml-js master --logging.level.com.synopsys.integration=DEBUG
+    displayName: 'Blackduck Scan'
```

Comment:
Build is failing, probably it's this indentation? Spaces are meaningful in YAML, right?